dashboard/app/views/levels/editors/fields/_validations.html.haml

```diff
@@ -5,4 +5,4 @@
 
 - content_for :body_scripts do
   %script{src: webpack_asset_path('js/levels/editors/fields/_validations.js'),
-        data: {validations: @level.level_data["validations"].to_json, levelname: @level.name, appname: @level.game&.app}}
+        data: {validations: (@level.properties.dig('level_data', 'validations') || []).to_json, levelname: @level.name, appname: @level.game&.app}}
```

Comment:
What a cool method `dig` is!

Reply:
Learned about it from @etaderhold in this PR! https://github.com/code-dot-org/code-dot-org/pull/56368/